scipy/stats/mstats_basic.py

```diff
@@ -549,6 +549,42 @@ def _spearmanr_2cols(x):
         return SpearmanrResult(rs, prob)
 
 
+def _kendall_p_exact(n, c):
+    # Exact p-value, see Maurice G. Kendall, "Rank Correlation Methods" (4th Edition), Charles Griffin & Co., 1970.
+    if n <= 0:
+        raise ValueError
```

Comment:
```suggestion
        raise ValueError('n ({n}) must be positive')
```
If the user could get here, I would want to provide how `n` relates to their input, but empty `x` and `y` is already checked, so this is fine.